docker/images/e2e/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM marsha:dev as marsha
```

Comment:
Why not install these dependencies in the dev image directly?

Reply:
Our containers are built on Debian 10. Playwright is based on Ubuntu 20. So a dedicated build is needed anyway.
Those dependencies would be useless in our dev image, as browsers won't be available.

Reply:
IC. Ok then.